### PR TITLE
Ignore updates

### PR DESCRIPTION
**Pull request by kbjarkefur**

### CODE REVIEW

File: GitHub-resources/DIME-GitHub-Templates/gitignore_template.txt [hunk 1]
```diff
@@ -48,6 +47,10 @@
 !/**/*.R
 !/**/*.Rmd
 !/**/*.Rproj
+# Still ignore shiny app meta files
+**/rsconnect/**/*.dcf
+# Still ignore user file for R projects
+.Rproj.user
```
Comment (kbjarkefur) [inline GitHub-resources/DIME-GitHub-Templates/gitignore_template.txt]: @luizaandrade does this look right to you? These should both remained ignored, right?

File: GitHub-resources/DIME-GitHub-Templates/gitignore_template.txt [hunk 2]
```diff
@@ -56,6 +59,9 @@
 # Python
 !/**/*.py
 !/**/*.ipynb
+!requirements.txt
+!Pipenv
+!Pipfile.lock
```
Comment (kbjarkefur) [inline GitHub-resources/DIME-GitHub-Templates/gitignore_template.txt]: @rcarroll901 just want to confirm with you that both of these should be un-ignored

Comment (rcarroll901) [reply GitHub-resources/DIME-GitHub-Templates/gitignore_template.txt]: @kbjarkefur Yes, they should both be committed to VCS. Just change "!Pipenv" to "!Pipfile"

File: GitHub-resources/DIME-GitHub-Templates/gitignore_template.txt (hunk 1 above)
Comment (luizaandrade) [reply GitHub-resources/DIME-GitHub-Templates/gitignore_template.txt]: yes, that's right! sorry for the confusion

File: GitHub-resources/DIME-GitHub-Templates/gitignore_template.txt [hunk 3]
```diff
@@ -48,6 +47,10 @@
 !/**/*.R
 !/**/*.Rmd
 !/**/*.Rproj
+# Still ignore shiny app meta files
+**/rsconnect/**/*.dcf
```
Comment (luizaandrade) [inline GitHub-resources/DIME-GitHub-Templates/gitignore_template.txt]:
```suggestion

```

**Review by luizaandrade — APPROVED** [review]

File: GitHub-resources/DIME-GitHub-Templates/gitignore_template.txt (hunk 2 above)
Comment (kbjarkefur) [reply GitHub-resources/DIME-GitHub-Templates/gitignore_template.txt]: Resolving as this line is now outdated

File: GitHub-resources/DIME-GitHub-Templates/gitignore_template.txt [hunk 4]
```diff
@@ -56,6 +57,9 @@
 # Python
 !/**/*.py
 !/**/*.ipynb
+!/**/requirements.txt
+!/**/Pipenv
+!/**/Pipfile.lock
```
Comment (kbjarkefur) [inline GitHub-resources/DIME-GitHub-Templates/gitignore_template.txt]: @rcarroll901 just want to confirm with you that both of these should be un-ignored

Comment (rcarroll901) [reply GitHub-resources/DIME-GitHub-Templates/gitignore_template.txt]: !Pipenv needs to be changed to !Pipfile. Otherwise, looks great.

Comment (kbjarkefur) [reply GitHub-resources/DIME-GitHub-Templates/gitignore_template.txt]:
```suggestion
!/**/Pipfile
!/**/Pipfile.lock
```
Like this? Hit commit if correct

**Pull request merged**